### PR TITLE
Release changelog for 2.41

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

**Comment by kyrofa** [comment]
Looks like Sentry issues in autopackagetests.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+  * storeapi: properly handle lacking permission for channel (#2050)
+  * many: add snapcraftctl set-version (#2063)
+  * storeapi: handle 500 error response when releasing snap (#2059)
+  * states: track override scriptlets (#2068)
+  * many: add snapcraftctl set-grade (#2067)
+  * meta: validate extracted and scriptlet metadata (#2073)
+
+  [ Christian Dywan ]
+  * options: introduce Project and ProjectInfo (#1995)
+  * lxd: friendly errror with suggestions if network is broken (#1930)
```
Comment (kyrofa) [inline debian/changelog]: errrrrrrrrrror

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,57 @@
+snapcraft (2.41) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * package: ensure all relevant files are in for sdist (#2060)
+  * errors: enable sending tracebacks to sentry (#1961) (Closes: #1918)
```
Comment (kyrofa) [inline debian/changelog]: Probably don't want the Closes in here, do we?

Comment (sergiusens) [reply debian/changelog]: I was adding them, but apparently we all need to use `Closes`

**Comment by sergiusens** [comment]
@kyrofa yeah, I pop a fix in a bit

**Pull request merged**